lib/Profile.dart

```diff
@@ -0,0 +1,227 @@
+import 'dart:io';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:image_picker/image_picker.dart';
+
+class Profile extends StatefulWidget {
+  Profile({Key key}) : super(key: key);
+  @override
+  _ProfileState createState() => _ProfileState();
+}
+
+class _ProfileState extends State<Profile> {
+  File _image;
```

Comment:
Does not persist once changed, meaning that an entirely new Profile Widget (with a new state) might be created every time you re-enter the page. Let's see if we can change that somehow.

Reply:
Removed all the profile pic stuff for now, will work on this for the actual functionality connected to backend